docs/content/version-reporting.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Anonymous Version Reporting
+kind: operations
+weight: 50
+---
+
+This document provides details about OPA's anonymous version reporting feature.
+
+## Overview
+
+OPA supports anonymous version reporting wherein OPA would reach out to an external service and share its version 
+information. This feature is only applicable to the `opa run` and `opa version` commands.
+
+In case of the `opa run` command, this feature will be **ON by-default** and can be easily disabled by specifying
+the `--incognito` flag. When OPA is started in either the `server` or `repl` mode, OPA would call the external service
+on a best-effort basis and share the version it's running. The time taken to execute the remote call and process the
+subsequent response from the external service would not delay OPA's start-up.
+
+In case of the `opa version` command, this feature can be enabled by specifying the `--check` or `-c` flag.
+
+## External Service
+
+OPA uploads its version information at [telemetry.openpolicyagent.org](https://telemetry.openpolicyagent.org). This is
+the default external service and can be made configurable in the future.
+
+The external service checks the OPA version reported by a remote OPA client and if it doesn't match the latest
+released OPA version, the external service responds with information about the latest OPA release. This information includes
+a link to download the latest OPA version, release notes etc.
+
+The external service generates a report documenting the number of different OPA versions in deployment and makes this
+report publicly available.
+
+## Benefits
+
+* OPA's anonymous version reporting feature provides users with up-to-date information about new OPA versions while
+still executing the familiar OPA `run` and `version` commands. It helps users stay abreast of OPA's latest capabilities
+and hence empowers them to make informed decisions while upgrading their OPA deployments.
+
+* Both new and experienced OPA users can leverage the publicly available information about the deployed OPA versions
+to decide which OPA version is the most suitable for their deployment environments. This information should also give
+users more confidence when they deploy OPA in production.
+
+* Information gathered about deployed OPA versions can also help in making better decisions about deprecating older OPA versions.
```

Comment:
I'd add a point that this gives the OPA maintainers and the CNCF more information about OPA usage and engagement.

Reply:
Added a point on this